modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -484,38 +508,33 @@ However, the `aria-placeholder` attribute is not a valid attribute for the body
-
-=== `sandbox_iframes` editor option is now defaulted to `true`.
+=== Optional Tooltip for Dialog Footer Toggle Buttons
+// #TINY-10672
 
-In {productname} 6.8.1, the xref:content-filtering.adoc#sandbox-iframes[sandbox iframes] editor option was introduced to allow iframes to be sandboxed by default when inserted into the editor.
+Previously, the `DialogFooterToggleButton` component required a tooltip property. This meant including a tooltip for all toggle buttons displayed in dialog footers, even if the information wasn't necessary.
 
-In {productname} 7.0, the default for `+sandbox_iframes+` will change from `false` to `true`, meaning that all `+iframe+` elements inserted into the editor will be given the `sandbox=""` attribute by default, preventing most actions, including scripting and same-origin access, which may break existing editor content or produce undesirable effects.
+As of {productname} 7.0, the tooltip property is now optional for `DialogFooterToggleButton`. This flexibility allows developers to omit the tooltip for buttons where it's not required, providing a cleaner and more focused user experience.
 
-To prevent any expected iframes from being sandboxed, we recommend adding the source domains of such iframes to the new xref:content-filtering.adoc#sandbox-iframes-exclusions[`+sandbox_iframes_exclusions+`] option list, and including the domains in the default list where necessary. To prevent all iframes from being sandboxed, set the option `+sandbox_iframes+` to `+false+` in your editor configuration.
+=== Changed the `media_url_resolver` option to use promises.
+// #TINY-9154
 
-For further details on the `+sandbox_iframes+` option, see the xref:content-filtering.adoc#sandbox-iframes[the content filtering options], or refer to the xref:security.adoc#sandbox-iframes[security guide], or the link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-sandbox_iframes-option-that-controls-whether-iframe-elements-will-be-added-a-sandbox-attribute-to-mitigate-malicious-intent[{productname} 6.8.1 release notes].
+In {productname} 7.0, the `media_url_resolver` has been refactored to use promises instead of callbacks. This change brings standardization to the code base and improves error handling.
 
-=== Remove the height field from the `table` plugin cell dialog. The `table` plugin row dialog now controls the row height by setting the height on the `tr` element, not the `td` elements.
-//# TINY-10617
+// TODO: demo to show the function changing from `(data, resolve, reject) => void` to `(data) => Promise<result>`
```

Comment:
- [ ] add demo to support TINY-9154

Reply:
outdated, i linked this to the xref:media.adoc#media_url_resolver[media_url_resolver] page, and added entry to the migration guide.